src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -24,30 +33,119 @@ For more info on Sentry CLI configuration visit the [Sentry CLI configuration do
 
 </Note>
 
-Make sure Sentry CLI is configured for your project. You can use a `.sentryclirc` file or environment variables:
-
 <Note>
 
 Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.
 
 </Note>
 
-```ìni {filename:.sentryclirc}
-[auth]
-token=your-auth-token
+## Upload with Debug ID (Preferred)
 
-[defaults]
-org=___ORG_SLUG___
-project=___PROJECT_SLUG___
+Debug IDs are a new feature we have released to account for most of the problems that users were incurring in with the release-based system. Since having events not symbolicated is big problem for our users, we designed Debug IDs to provide a reliable system that would solve the problems people were incurring.
+
+### 1. Prepare Your Bundle
+
+In order to use Debug IDs you will have to bundle your code with the `sourcesContent` field in the source map. This is required by Sentry when processing your events and will lead to problems in the case of absence since the context lines will not be computed.
+
+<Note>
+
+The generation of the `sourcesContent` will depend on your bundler configuration, therefore we reccomend you looking at the official docs.
+
+</Note>
+
+### 2. Enable The Debug IDs In Sentry CLI
+
+Since Debug IDs are currently opt-in only, you should enable the following environment variable, which will tell Sentry CLI to use the new Debug IDs system for uploading your artifacts.
+
+```bash
+export SENTRY_FORCE_ARTIFACT_BUNDLES=1
 ```
 
+### 3. Inject Debug IDs Using Sentry CLI
+
+Debug IDs require an additional but very important step, the injection. In this step, [`sentry-cli`](/product/cli/) will go over all the minified sources and their respective source maps and it will start adding custom metadata that is required by Sentry for correctly symbolicating the event.
```

Comment:
As a user, I assume every step is important unless specified otherwise, so let's remove the first sentence from this paragraph.